docs/kane-ai-using-datasets.md

```diff
@@ -0,0 +1,95 @@
+---
+id: kane-ai-using-datasets
+title: KaneAI - Using Datasets on Test Manager
+hide_title: false
+sidebar_label: Datasets
+description: Learn how to use datasets in the KaneAI test cases
+keywords:
+  - lambdatest automation
+  - lambdatest kaneai
+  - kaneai scroll elements
+  - kaneai sidebar scroll
+url: https://www.lambdatest.com/support/docs/kane-ai-using-datasets/
+site_name: LambdaTest
+slug: kane-ai-using-datasets/
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+import {YOUR_LAMBDATEST_USERNAME, YOUR_LAMBDATEST_ACCESS_KEY} from "@site/src/component/keys";
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "KaneAI Datasets",
+          "item": "https://www.lambdatest.com/support/docs/kane-ai-using-datasets/"
+        }]
+      })
+    }}
+></script>
+Datasets in KaneAI-LambdaTest allow you to manage and reuse input values for test cases efficiently. This guide explains how to create, edit, and manage these data sets while utilizing parameters to run test cases with a variety of inputs, improving flexibility and reducing manual data entry.
+
+## Step 1: Accessing Parameters and Datasets
```

Comment:
@amanchopra1905 - Please remove the steps structure from all new docs you are creating. It loses the sense of being a support doc and becomes instructions only. Repeating this again

Reply:
Mark the section as "Accessing Datasets within a Project"